cartographer_rviz/cartographer_rviz/submaps_display.cc

```diff
@@ -66,6 +66,9 @@ SubmapsDisplay::SubmapsDisplay() : tf_listener_(tf_buffer_) {
       "All", true,
       "Whether submaps from all trajectories should be displayed or not.",
       trajectories_category_, SLOT(AllEnabledToggled()), this);
+  fade_out_start_distance_in_meters_ = new ::rviz::FloatProperty(
+      "Fade-out start distance in m", 1.,
+      "Fade out submaps that are farther away than this in z-direction.", this);
```

Comment:
You could mention the units here if you want to: `Distance in meters in z-direction beyond which submaps will start to fade out.`?